samples/RuleMetadata.sarif

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "properties": {
+    "comment": "This sample demonstrates rule metadata and the connection between results and the metadata for the corresponding rules."
+  },
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SarifSamples",
+          "rules": [
+            {
+              "id": "SMP0001",
+              "name": "InvalidUri",
+              "defaultConfiguration": {
+                "level": "error"
+              },
+              "shortDescription": {
+                "text": "URI-valued properties must contain valid URIs."
```

Comment:
>text [](start = 17, length = 4)

Suggest adding markdown descriptive text, and messages as well. Doesn't bloat the sample overmuch and will be useful when someone gets around to, say, display rule descriptions in a markdown control. We already have a viewer (Jeff's) this will display markdown messages if available).